qi-lib/flow/core/compiler/deforest/fusion.rkt

```diff
@@ -0,0 +1,76 @@
+#lang racket/base
```

Comment:
Splitting this file out should probably be with the relevant commit that
introduces it, but that might be challenging now, so I'm ok with a refactor
commit that includes only this extraction.